sdk/communication/Azure.Communication.Identity/src/CommunicationIdentityClient.cs

```diff
@@ -111,15 +150,23 @@ public virtual Response<CommunicationUserIdentifier> CreateUser(CancellationToke
             }
         }
 
-        /// <summary>Asynchronously creates a new <see cref="CommunicationUserIdentifier"/>.</summary>
+        /// <summary>Creates a new <see cref="CommunicationUserIdentifier"/>.</summary>
         /// <param name="cancellationToken">The cancellation token to use.</param>
-        public virtual async Task<Response<CommunicationUserIdentifier>> CreateUserAsync(CancellationToken cancellationToken = default)
+        public virtual Response<CommunicationUserIdentifier> CreateUser(CancellationToken cancellationToken = default)
+        {
+            return CreateUser(default, cancellationToken);
```

Comment:
nit: we can use expression bodies for single line method bodies

Reply:
Done